src/campaign/javascript/app.js

```diff
@@ -16,4 +16,72 @@ function activeHeader() {
   } else {
     header.className = "header";
   }
-}
\ No newline at end of file
+}
+
+function closePlayer(player, fullscreen) {
+  if (fullscreen) {
+    player.fullscreen.exit();
+  }
+  player.stop();
+}
+
+
+var plyr1 = document.getElementById('player-1'),
+  plyr2 = document.getElementById('player-2');
+
+if (plyr1 && plyr2) {
```

Comment:
this needs refactoring into a module to fit the rest, will accept for now but will need to be tidied up

Reply:
Agreed